lib/ansible/playbook/task.py

```diff
@@ -506,6 +591,15 @@ def _get_parent_attribute(self, attr, extend=False, prepend=False):
                         value = self._extend_value(value, parent_value, prepend)
                     else:
                         value = parent_value
+
```

Comment:
this change can probably go away entirely if you ditch the `_validate` method as proposed above